frontend/src/metabase/components/EmptyState.css

```diff
@@ -0,0 +1,15 @@
+:local(.emptyState) {
+    composes: text-centered text-brand-light from "style";
```

Comment:
If we're only using preexisting classnames from our core css in cases like the ones in this file we should just use those inline in the component itself. Using CSS modules in a case like this basically ends up becoming duplicate code.